projects/plugins/jetpack/modules/search/instant-search/components/search-filters.scss

```diff
@@ -68,3 +68,8 @@
 		width: auto;
 	}
 }
+
+.jetpack-instant-search__search-static-filter-list {
+  font-size: 0.875rem;
```

Comment:
You'll want to use tabs instead of spaces here, to avoid unwanted changes the next time someone edits that file and transforms all spaces into tabs on save.

Reply:
Good catch, fixed.